web/src/common/ContextLayerButton.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { HelpButton } from "../common";
+
+  export let label: string;
+  export let show = false;
+</script>
+
+<button
+  style="display: flex; align-items: center; justify-content: leading; gap: 8px; text-align: left;"
+  class="secondary"
+  on:click={() => (show = !show)}
+>
+  <input type="checkbox" bind:checked={show} />
+  {label}
+  {#if $$slots.help}
+    <HelpButton color="var(--pico-secondary-inverse)">
```

Comment:
The help button nested inside the layer button means that when you click help, it toggles the layer on and off. This feels unintuitive to me, but happy to go with this for now and discuss/iterate later. Maybe just ending the outer `<button>` after `{label}` would be enough, I'll try later.